src/features/broker-protection/actions/fill-form.js

```diff
@@ -54,5 +73,38 @@ function setValueForInput (el, val) {
     events.forEach((ev) => el.dispatchEvent(ev))
     el.blur()
 
-    return true
+    return { result: true }
+}
+
+/**
+ * @param element
+ * @return {{result: true}|{result: false, error: string}}
+ */
+function setImageUpload (element) {
+    const base64PNG = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mP8/B8AAusB9VF9PmUAAAAASUVORK5CYII='
+    try {
+        // Convert the Base64 string to a Blob
+        const binaryString = window.atob(base64PNG)
+
+        // Convert binary string to a Typed Array
+        const length = binaryString.length
+        const bytes = new Uint8Array(length)
+        for (let i = 0; i < length; i++) {
+            bytes[i] = binaryString.charCodeAt(i)
+        }
+
+        // Create the Blob from the Typed Array
+        const blob = new Blob([bytes], { type: 'image/png' })
+
+        // Create a DataTransfer object and append the Blob
+        const dataTransfer = new DataTransfer()
+        dataTransfer.items.add(new File([blob], 'id.png', { type: 'image/png' }));
```

Comment:
I don't think it's necessary now, but we may want to consider randomizing the filename at some point in the future.

Reply:
agreed - let's follow up as needed.